components/scream/src/physics/p3/scream_p3_interface.hpp

```diff
@@ -13,7 +13,7 @@ extern "C"
 
 // Fortran routines to be called from C
 void p3_init_f90 ();
-void p3_standalone_init_f90 (Real* q, Real* T, Real* zi, Real* pmid, Real* dpres,
+void p3_standalone_init_f90 (Real* q, Real* t_atm, Real* zi, Real* pmid, Real* dpres,
```

Comment:
In some cases I see T -> t_atm and sometimes T -> T_atm.  is there a distinction between the two options?  If not, should we commit to uppercase or lowercase "T" consistently throughout?

Reply:
this is the argument variables, it use T_atm(t_atm), so I choose t_atm to differ it from T_atm inside the function.

Reply:
I thought we used "_" at the end where we had an argument we wanted to convert into a Pack later, so "T_atm(T_atm_)" would be the typical approach... but I'm confused why other arguments here don't do that. What does zi do to convert from argument to local Pack variable, for example? I'm confused because I don't see any argument assignment to packs in this function... I'm questioning your choice because I'd really like T_atm to be identical everywhere rather than switching cases for reasons that might be important but aren't immediately obvious...